.gitignore

```diff
@@ -2,3 +2,4 @@
 coverage/
 node_modules/
 npm-debug.log
+.DS_Store
```

Comment:
You can remove this; setup a global ignore for your system if you need to ignore system-specific files.

Reply:
ok